### ISSUE TITLE
[Issue] Error with sqlx-core when building Nym

### ISSUE THREAD

**Issue by lcheylus** [nymtech/nym#1818]
When I tried to build Nym on Linux Debian/testing (amd64) architecture, I have an error with `sqlx-core` library.

**Steps to Reproduce**
Build of Nym according to https://nymtech.net/docs/stable/run-nym-nodes/build-nym/

```
$ git clone https://github.com/nymtech/nym.git nym.git
$ cd nym.git
$ git checkout release/v1.1.0
$ cargo build --release
(...)
Updating crates.io index
    Updating git repository `https://github.com/mmsinclair/wasm-timer`
    Updating git repository `https://github.com/nymtech/sphinx`
    Updating git repository `https://github.com/neacsu/cosmos-rust`
    Updating git submodule `https://github.com/cosmos/cosmos-sdk.git`
    Updating git submodule `https://github.com/cosmos/ibc-go`
    Updating git submodule `https://github.com/CosmWasm/wasmd.git`
    Updating git repository `https://github.com/jstuczyn/bls12_381`
    Updating git repository `https://github.com/lawliet89/rocket_cors`
error: failed to select a version for `sqlx-core`.
    ... required by package `sqlx v0.6.2`
    ... which satisfies dependency `sqlx = "=0.6.2"` of package `nym-network-requester v1.1.0 (/home/fox/dev/Tools/nym.git/service-providers/network-requester)`
versions that meet the requirements `=0.6.2` are: 0.6.2

the package `sqlx` depends on `sqlx-core`, with features: `sqlx` but `sqlx-core` does not have these features.


failed to select a version for `sqlx-core` which could resolve this conflict
```

**Build infos**

- Linux Debian/testing on amd64
- Rust version 1.62.1
```
$ rustc -V
rustc 1.62.1
```

**Comment by tommyv1987** [comment]
HI @lcheylus can you please try our latest release branch which will be `release/v1.1.1`, also can you update your rust version?

Currently, I'm running rust version - `rustc 1.65.0 (897e37553 2022-11-02)` and let me know how you get on.

**Comment by lcheylus** [comment]
Problem fixed with `release/v1.1.1` and using Rust v1.65 => build OK
